cypress/support/e2e.js

```diff
@@ -18,3 +18,19 @@ import './commands';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+Cypress.Commands.add('login', (username, password) => {
+  cy.request({
+    method: 'POST',
+    url: 'https://demoqa.com/Account/v1/Login',
+    body: {
+      userName: username,
+      password
+    }
+  }).then((response) => {
+    cy.setCookie('token', response.body.token);
+    cy.setCookie('userID', response.body.userId);
+    cy.setCookie('expires', response.body.expires);
+    cy.setCookie('userName', response.body.username);
+  });
+});
```

Comment:
I can't find the usage of this code snippet. 
It would be better to store Custom commands in the ```commands.js```